packaging/setup/plugins/ovirt-engine-setup/ovirt-engine/config/ansible_runner.py

```diff
@@ -0,0 +1,146 @@
+#
```

Comment:
As we are copying the whole project directory, then this is not needed:

1. /var/lib/ovirt-engine is already handled within spec file, we can also add the creation of ansible-runner-project subdirectory there: https://github.com/oVirt/ovirt-engine/blob/master/ovirt-engine.spec.in#L809
2. In AnsibleCommandConfig we can actually create specific project pre run:
  - Create /var/lib/ovirt-engine/ansible-runner-project/<RUN UUID> along with env and inventory subdirectories
  - Create a link from /var/lib/ovirt-engine/ansible-runner-project/<RUN UUID>/project to /usr/share/ovirt-engine/ansible-runner-service-project/project
  - Create link to engine private ssh key

That way the whole management of specific project directory per run will be on the same place

Reply:
done